presentation/src/main/java/me/androidbox/presentation/agenda/viewmodel/AgendaViewModel.kt

```diff
@@ -50,6 +52,7 @@ class AgendaViewModel @Inject constructor(
         fetchAgendaItems(ZonedDateTime.now())
         syncAgendaItems()
         syncFullAgendaItems()
+        synAgendaTasksReminders()
```

Comment:
What's the difference between 

syncAgendaItems()
syncFullAgendaItems()
synAgendaTasksReminders()